tests/test_soft.nf.test.snap

```diff
@@ -76,15 +87,19 @@
                 "app.R:md5,bedcfc45b6cdcc2b8fe3627987e2b17a",
                 "versions.yml:md5,27cf350d1f743e60e64b4434cf6d4687",
                 "normalised.annotation.tsv:md5,40300ae222ae35fa54daf10a1b86e830",
-                "phenotype_uninvolved_lesional_study.limma.results.tsv:md5,2cfa653fffb034be4b82abf306c87bc5",
-                "phenotype_uninvolved_lesional_study.limma.results_filtered.tsv:md5,6a198c35b193bab0b3148d147c7b08e3",
-                "normalised.matrix.tsv:md5,230f87f91c0c1a7e2ab354200c4b978c"
+                "phenotype_uninvolved_lesional.limma.results.tsv:md5,2cfa653fffb034be4b82abf306c87bc5",
```

Comment:
Why have some output file names changed?

Reply:
This was a temporary change after merging the dev branch from the GitHub editor. The current tests do not change output names and do not differ to previous tests from dev (except ofc the new yaml tests that were added).